sn_node/src/node.rs

```diff
@@ -520,48 +519,7 @@ impl Node {
     }
 
     fn handle_node_cmd(&self, cmd: Cmd) {
```

Comment:
lets also remove this and the corresponding `CmdRequestReceived` if we're notneeding those anymore 🤔